torch/nn/_intrinsic/qat/modules/conv_fused.py

```diff
@@ -0,0 +1,255 @@
+        self.beta = Parameter(torch.Tensor(out_channels))
+        self.affine = True
+        self.track_running_stats = True
+        self.register_buffer('running_mean', torch.zeros(out_channels))
+        self.register_buffer('running_var', torch.ones(out_channels))
+        self.register_buffer('num_batches_tracked', torch.tensor(0, dtype=torch.long))
+        self.observer = activation_fake_quant()
+        self.weight_fake_quant = weight_fake_quant()
+        self.reset_bn_parameters()
+
+    def reset_running_stats(self):
+        self.running_mean.zero_()
+        self.running_var.fill_(1)
+        self.num_batches_tracked.zero_()
+
+    def reset_bn_parameters(self):
+        self.reset_running_stats()
+        init.uniform_(self.gamma)
+        init.zeros_(self.beta)
+
+    def reset_parameters(self):
+        super(ConvBn2d, self).reset_parameters()
+        # A hack to avoid resetting on undefined parameters
+        if hasattr(self, 'gamma'):
+            self.reset_bn_parameters()
+
+    def enable_fake_quant(self):
+        self.observer.enable()
+        self.weight_fake_quant.enable()
+        return self
+
+    def disable_fake_quant(self):
+        self.observer.disable()
+        self.weight_fake_quant.disable()
+        return self
+
+    def _forward(self, input):
+        # exponential_average_factor is self.momentum set to
+        # (when it is available) only so that if gets updated
+        # in ONNX graph when this node is exported to ONNX.
+        if self.momentum is None:
+            exponential_average_factor = 0.0
+        else:
+            exponential_average_factor = self.momentum
+
+        if self.training and self.track_running_stats:
+            # TODO: if statement only here to tell the jit to skip emitting this when it is None
+            if self.num_batches_tracked is not None:
+                self.num_batches_tracked += 1
+                if self.momentum is None:  # use cumulative moving average
+                    exponential_average_factor = 1.0 / float(self.num_batches_tracked)
+                else:  # use exponential moving average
+                    exponential_average_factor = self.momentum
+
+        # we use running statistics from the previous batch, so this is an
+        # approximation of the approach mentioned in the whitepaper, but we only
+        # need to do one convolution in this case instead of two
+        running_std = torch.sqrt(self.running_var + self.eps)
+        scale_factor = self.gamma / running_std
+        scaled_weight = self.weight * scale_factor.reshape([-1, 1, 1, 1])
+        conv = self.conv2d_forward(input, self.weight_fake_quant(scaled_weight))
+
+        if self.training and not self.freeze_bn:
+            # recovering original conv to get original batch_mean and batch_var
+            conv_orig = conv / scale_factor.reshape([1, -1, 1, 1])
+            batch_mean = torch.mean(conv_orig, dim=[0, 2, 3])
+            if self.bias is not None:
```

Comment:
Please remove if self.bias and corresponding code here, self.bias should be None.